app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -29,7 +27,15 @@ class ProfileListAdapter(
     ListAdapter<Profile, ProfileListAdapter.ViewHolder>(
         ProfileDiffCallbacks()
     ) {
-
+    companion object {
```

Comment:
shift the companion object to Details Fragment as it does not make sense to put this code here.
Also, change the TAG value to something relevant.

Reply:
@rob729 I have shifted companion object to Details Fragment.
I have removed "TAG" as it was an extra variable that was not being used anywhere.